internal/tokenizer/flex/tokenize_c.go

```diff
@@ -0,0 +1,71 @@
+package flex
+
+// #include <stdlib.h>
+// #include "linguist.h"
+// #include "lex.linguist_yy.h"
+// int linguist_yywrap(yyscan_t yyscanner) {
+// 	return 1;
+// }
+import "C"
+import "unsafe"
+
+const maxTokenLen = 32 // bytes
+
+// TokenizeFlex implements tokenizer by calling Flex generated code from linguist in C
+// This is a transliteration from C https://github.com/github/linguist/blob/master/ext/linguist/linguist.c#L12
+func TokenizeFlex(content []byte) []string {
+	var buf C.YY_BUFFER_STATE
```

Comment:
Merge into a single `var` block

Reply:
I know what you mean, but would prefer to keep it as close to original as possible as this is almost [line-by-line transliteration from C](https://github.com/github/linguist/blob/master/ext/linguist/linguist.c#L12)

Reply:
The key word is "almost" :) Unless it's machine-generated, I think it makes sense to adhere to Go conventions where possible. It's not a bit change anyway. Still, it's optional, so feel free to ignore.